integration-templates/anrok/actions/create-ephemeral-transaction.ts

```diff
@@ -0,0 +1,66 @@
+import type { NangoAction, TransactionActionResponse, SuccessTransaction, Transaction, AnrokTransactionData } from '../../models';
+
+import type { AnrokResponse } from '../types';
+import { mapFees } from '../mappers/fees.js';
+
+export default async function runAction(nango: NangoAction, rawInput: Transaction[]): Promise<TransactionActionResponse> {
+    const response: TransactionActionResponse = {
+        succeeded: [],
+        failed: []
+    };
+
+    const input = Array.isArray(rawInput) ? rawInput : [rawInput];
```

Comment:
would the input validation not fail if `rawInput` is actually not an array?

Reply:
it would when validation is active but it isn't blocking currently, just warns